### ISSUE TITLE
Public Test Results: Movement Analysis [v0.1.0-alpha]

### ISSUE THREAD

**Issue by JuiceWarrior** [Teeyenoh/zephis#4]
Movement is currently very bare-bones - any change in direction is often not registered until the block has stopped moving creating an immediate friction with the player.

As a result, movement feels very clunky and uninteresting, it's almost a chore waiting for the game to emulate any changes to the block's movement direction the player inputs.

**Comment by Teeyenoh** [comment]
What would you suggest? I don't fully understand what you mean, but I think I get the idea. Would it be better if movements issued a few seconds BEFORE the end of a tile were counted as well?

**Comment by JuiceWarrior** [comment]
This is a problem that exists because of how movement is registered. Remember that discussion we had about shrinking the 'grid' of movement so movements were registered more frequently? It's the same problem here.

**Comment by JuiceWarrior** [comment]
Movements simply take too long to register, hence lose fluidity as a result

**Comment by JuiceWarrior** [comment]
You could either fix this by speeding up the movement speed, or increasing the frequency of points where key presses are registered.

**Comment by JuiceWarrior** [comment]
![15226342_1324192300965050_1041864000_n](https://cloud.githubusercontent.com/assets/23152872/23314903/bf7da9b6-fabb-11e6-8c09-fd446d7de424.jpg)

If you remember, I used this image to help visualise my point of a potential solution for this problem.

**Comment by Teeyenoh** [comment]
It should be greatly improved, if not resolved, in v0.1.1 which you can now test

**Comment by JuiceWarrior** [comment]
Nah, it's still too slow. For instance, try pressing the forward and backward keys over and over again. You'll get what I mean.

**Comment by JuiceWarrior** [comment]
The square will typically just move in one direction, regardless of your inputs

**Comment by Teeyenoh** [comment]
You must be playing the wrong version :-) the currently uploaded v0.1.1 doesn't actually run!

**Comment by JuiceWarrior** [comment]
I edited the bin file to open 0.1.1 jar though :P

**Comment by Teeyenoh** [comment]
I'll look more at movement later, it feels satisfactory now

**Comment by JuiceWarrior** [comment]
Update [v0.1.2-alpha]: Movement is still too slow!
